src/main/java/executor/service/model/Step.java

```diff
@@ -1,9 +1,24 @@
 package executor.service.model;
 
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+
 import java.util.Objects;
 
+@Entity
+@Table(name = "steps")
 public class Step {
```

Comment:
Persistence object and DTO are 2 different things.

Reply:
I agree with you and will fix it.